modules/cnf-topology-aware-lifecycle-manager-operator-update.adoc

```diff
@@ -53,17 +53,54 @@ spec:
 <1> The index image URL contains the desired Operator images. If the index images are always pushed to the same image name and tag, this change is not needed.
 <2> Set how frequently the Operator Lifecycle Manager (OLM) polls the index image for new Operator versions with the `registryPoll.interval` field. This change is not needed if a new index image tag is always pushed for y-stream and z-stream Operator updates. The `registryPoll.interval` field can be set to a shorter interval to expedite the update, however shorter intervals increase computational load. To counteract this, you can restore `registryPoll.interval` to the default value once the update is complete.
 <3> Last observed state of the catalog connection. The `READY` value ensures that the `CatalogSource` policy is ready, indicating that the index pod is pulled and is running. This way, {cgu-operator} upgrades the Operators based on up-to-date policy compliance states.
++
+[NOTE]
+====
+After a catalog source update, it takes time for OLM to update the subscription status. This can mean that the status of the subscription policy may continue to show as compliant when TALM decides whether remediation is needed. As a result the Operator specified in the subscription policy does not get upgraded. To mitigate this issue, the `lastObservedState: READY` specification reduces the time between the completion of the catalog source policy remediation and the update of the subscription status. 
+
+However, if the issue persists, you can add another catalog source configuration to the `PolicyGenTemplate` resource with the same specifications, except for the following changes: 
+
+[source,yaml]
+----
+- fileName: DefaultCatsrc.yaml
+  # Contains original catalog source configuration
+  ...
+- fileName: DefaultCatsrc.yaml 
+      remediationAction: inform
+      policyName: "operator-catsrc-policy"
+      metadata:
+        name: redhat-operators-v2 <1>
+      spec:
+        displayName: Red Hat Operators Catalog v2 <2>
+        image: registry.example.com:5000/olredhat-operators:<version> <3>
+...
+----
+<1> Update the name for the new configuration.
+<2> Update the display name for the new configuration.
+<3> Update the index image URL. This `fileName.spec.image` field overrides any configuration in the `DefaultCatsrc.yaml` file.
+
+Update the `Subscription` resource to point to the new configuration for Operators that require an update:
```

Comment:
This is the first that I'm seeing a Subscription resource mentioned in the module (and assembly). Should there be some information on how to get/update this resource? The YAML doesn't have a name or namespace defined, which I think we would need to know?

Reply:
The subscriptions are in the same repo as the `defaultcatsrc.yaml` file. As this is about updating Operators, the subscriptions should all already be present, it's more of a Day 2 operation. It can be a subscription to any Operator but I've just included some sample values.